### PR TITLE
PIO-164 Fix dependency and package name in examples

### PR DESCRIPTION
**Pull request by takezoe**

### CODE REVIEW

File: client/pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@ limitations under the License.
   <parent>
     <groupId>org.apache.predictionio</groupId>
     <artifactId>predictionio-sdk-java</artifactId>
-    <version>0.13.1-SNAPSHOT</version>
+    <version>0.13.0</version>
```
Comment (takezoe) [inline client/pom.xml]: It looks to be updated by maven-release-plugin automatically. I'm not sure whether I should rollback by hand in this case.

**Comment by takezoe** [comment]
@dszeto Could you take a look? I think it's nice if we can add some tests to check compilation of examples. But I'm not sure it's worth or not because Java SDK isn't updated so frequently.

File: client/pom.xml (hunk 1 above)
Comment (dszeto) [reply client/pom.xml]: I don't think we need to rollback the version, as the release plugin already created a release tag `v0.13.0` for the commit that has `0.13.0` as the release version number.

**Comment by dszeto** [comment]
Other than version numbers, I think the rest looks good. Thanks @takezoe !

**Comment by takezoe** [comment]
@dszeto Thanks for your review. I reverted version numbers.

By the way, do you mean our next release (rc2) will be 0.13.1? If so, I guess that versions in examples's `pom.xml` need to be updated to `0.13.1` by hand. Or can maven-release-plugin retag `0.13.0`?

**Comment by dszeto** [comment]
I will give a try and see if the plugin can retag `0.13.0`. Thank you!

**Pull request closed without merging**